pkgs/development/python-modules/md2pdf/default.nix

```diff
@@ -0,0 +1,57 @@
+{
+  buildPythonPackage,
+  docopt,
+  fetchFromGitHub,
+  lib,
+  markdown2,
+  pytest-cov-stub,
+  pytestCheckHook,
+  setuptools,
+  weasyprint,
+}:
+
+buildPythonPackage rec {
+  pname = "md2pdf";
+  version = "1.0.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "jmaupetit";
+    repo = "md2pdf";
+    rev = "refs/tags/${version}";
+    hash = "sha256-9B1vVfcBHk+xdE2Xouu95j3Hp4xm9d5DgPv2zKwCvHY=";
+  };
+
+  postPatch = ''
+    substituteInPlace setup.py \
```

Comment:
I'm wondering if you could use `pythonRemoveDeps`. To be checked.

Reply:
No. That only works for `dependencies`.

Reply:
Fair enough!